charts/opensearch/values.yaml

```diff
@@ -100,9 +100,11 @@ config:
 # Extra environment variables to append to this nodeGroup
 # This will be appended to the current 'env:' key. You can use any of the kubernetes env
 # syntax here
-extraEnvs: []
-#  - name: MY_ENVIRONMENT_VAR
-#    value: the_value_goes_here
+extraEnvs:
+  - name: PASSWORD
+    value: admin
+  - name: USERNAME
+    value: admin
 
```

Comment:
Hey @TheAlgo can we have the password field blank and allow user to pass in, if still left blank we can generate a random password in secret, we can use helm randAlphaNum(randAlphaNum 20 | b64enc ) string function and add it to a secret as follows
```
          - name: PASSWORD
            valueFrom:
              secretKeyRef:
                name: {{ template "opensearch.uname" . }}-credentials
```
This way the default password(admin:admin) is not added in charts by default and user can still override it passing in values.yaml file.

Reply:
Hey, just check this issue https://github.com/opensearch-project/security/issues/1576
adding directly as admin could miss from users end the action to change it, so which is why I have passed my above comment, but i'm ok as long as we add a comment above `- name: PASSWORD` saying that its recommended to change the password.

Reply:
That makes sense to add some more docstring and a line item in the docs as well. I can do that once @DandyDeveloper also completed reviewing this.

Reply:
@prudhvigodithi I have made the changes